Configuration/HLT/python/autoCondHLT.py

```diff
@@ -43,21 +43,21 @@
     'run2_hlt_Fake'          : ('run2_hlt_relval'         ,l1Menus['Fake']),
     'run2_hlt_Fake1'         : ('run2_hlt_relval'         ,l1Menus['Fake1']),
     'run2_hlt_Fake2'         : ('run2_hlt_relval'         ,l1Menus['Fake2']),
-    'run3_hlt_FULL'          : ('run2_hlt_relval'         ,l1Menus['FULL']),
-    'run3_hlt_GRun'          : ('run2_hlt_relval'         ,l1Menus['GRun']),
-    'run3_hlt_HIon'          : ('run2_hlt_relval'         ,l1Menus['HIon']),
-    'run3_hlt_PIon'          : ('run2_hlt_relval'         ,l1Menus['PIon']),
-    'run3_hlt_PRef'          : ('run2_hlt_relval'         ,l1Menus['PRef']),
+    'run3_hlt_FULL'          : ('run3_hlt'                ,l1Menus['FULL']),
+    'run3_hlt_GRun'          : ('run3_hlt'                ,l1Menus['GRun']),
+    'run3_hlt_HIon'          : ('run3_hlt'                ,l1Menus['HIon']),
+    'run3_hlt_PIon'          : ('run3_hlt'                ,l1Menus['PIon']),
+    'run3_hlt_PRef'          : ('run3_hlt'                ,l1Menus['PRef']),
 
     'run1_data_Fake'         : ('run1_data'               ,l1Menus['Fake']),
     'run2_data_Fake'         : ('run2_data_relval'        ,l1Menus['Fake']),
     'run2_data_Fake1'        : ('run2_data_relval'        ,l1Menus['Fake1']),
     'run2_data_Fake2'        : ('run2_data_relval'        ,l1Menus['Fake2']),
-    'run3_data_FULL'         : ('run2_data_relval'        ,l1Menus['FULL']),
-    'run3_data_GRun'         : ('run2_data_relval'        ,l1Menus['GRun']),
-    'run3_data_HIon'         : ('run2_data_promptlike_hi' ,l1Menus['HIon']),
-    'run3_data_PIon'         : ('run2_data_relval'        ,l1Menus['PIon']),
-    'run3_data_PRef'         : ('run2_data_relval'        ,l1Menus['PRef']),
+    'run3_data_FULL'         : ('run3_data'               ,l1Menus['FULL']),
+    'run3_data_GRun'         : ('run3_data'               ,l1Menus['GRun']),
+    'run3_data_HIon'         : ('run3_data'               ,l1Menus['HIon']),
+    'run3_data_PIon'         : ('run3_data'               ,l1Menus['PIon']),
+    'run3_data_PRef'         : ('run3_data'               ,l1Menus['PRef']),
```

Comment:
What is the reason to use run3_data instead of run3_data_prompt?

Reply:
No reason.. was an oversight on my part, sorry. Will change to `run3_data_prompt`.